pandas/_libs/window/aggregations.pyx

```diff
@@ -504,6 +504,7 @@ def roll_skew(ndarray[float64_t] values, ndarray[int64_t] start,
     )
     output = np.empty(N, dtype=float)
     min_val = np.nanmin(values)
+    values_copy = np.empty(N, dtype=float)
```

Comment:
why don't you simply copy values to begin with? then all of these additions are not needed

Reply:
Was not sure which would be faster. When we do a copy, we have to touch the values again in case of

``min_val - mean_val > -1e5``

Changed it